mino/minogrpc/server.go

```diff
@@ -386,52 +386,76 @@ type overlay struct {
 	router      router.Router
 	connMgr     session.ConnectionManager
 	addrFactory mino.AddressFactory
+	secret      interface{}
+	public      interface{}
 
 	// Keep a text marshalled value for the overlay address so that it's not
 	// calculated for each request.
 	meStr string
 }
```

Comment:
It wouldn't hurt to document the fields of the overlay struct, again, as a newcomer :)